lib/app.js

```diff
@@ -1,53 +1,44 @@
 (function (factory) {
   if (typeof define === 'function' && define['amd']) {
-    define(['knockout', './calculator', './todo-list'], factory);
+    define(['knockout'], factory);
   } else if (typeof require === 'function' && typeof exports === 'object' && typeof module === 'object') {
-    factory(require('knockout'), require('./calculator'), require('./todo-list'));
+    factory(require('knockout'));
   } else {
-    factory(ko, Calculator, ToDoList);
+    factory(ko);
   }
-} (function (ko, Calculator, ToDoList) {
+} (function (ko) {
 
   function App() {
-    let calculators = ko.observableArray();
-
+    let widgets = ko.observableArray();
```

Comment:
We use a single array of widgets. The array can contain both calculators and todo lists.